lib/src/painters/dotted_path_painter.dart

```diff
@@ -5,6 +5,7 @@ class DottedCropPathPainter extends CustomPainter {
   static const _dashWidth = 10.0;
   static const _dashSpace = 5.0;
   static const _strokeWidth = 4.0;
+  Color outlineColor;
```

Comment:
This can be final. I would also make this optional with Colors.white as fallback